test/diff/apply/out-pure.nix

```diff
@@ -69,12 +69,7 @@
     b = # Function call with comment
       mapAttrsToStringsSep "\n" mkSection attrsOfAttrs;
   }
-  [
-    (mapAttrsToStringsSep [
-      force
-      long
-    ] "\n" mkSection attrsOfAttrs)
-  ]
+  [ (mapAttrsToStringsSep [ force long ] "\n" mkSection attrsOfAttrs) ]
```

Comment:
nit: `force long` not very accurate any more

Reply:
Yep, seen that. Need to switch that over to comments or something else which is guaranteed to be multiline across all possible formats to prevent this in the future.